Remove never-supported MathML features

ddbeck opened the pull request:
This removes some stuff related to the removal of never-implemented features from BCD: https://github.com/mdn/browser-compat-data/pull/6958.

This is my first time cleaning up after a feature removal directly. Open to suggestions on how to handle this sort of thing going forward.

ddbeck committed:
Remove never-supported MathML element <mglyph>

ddbeck committed:
Remove never-supported MathML <math> attribute overflow

ddbeck committed:
Remove never-supported MathML element <mlabeledtr>

ddbeck committed:
Remove never-supported MathML <mo> attributes form, largeop

ddbeck committed:
Remove never-supported MathML <mspace> attribute linebreak

ddbeck committed:
Remove never-supported MathML <mystyle> attribute decimalpoint

ddbeck committed:
Remove never-supported MathML <mtable> attributes: alignmentscope, columnwidth, equalcolumns, equalrows, groupalign, minlabelspacing, side

ddbeck committed:
Remove never-supported MathML <mtd> attribute groupalign

https://github.com/mdn/browser-compat-data/pull/6958

ddbeck committed:
Remove never-supported MathML <mtr> attribute groupalign

https://github.com/mdn/browser-compat-data/pull/6958

chrisdavidmills reviewed (changes requested):
Looks mostly good @ddbeck ! Just a couple of points:

1. `<mglyph>` and `<mlabeledtr>` are both still listed on /en-US/docs/Web/MathML/Element
2. Did you use the correct tool for deleting documents (see https://github.com/mdn/content/blob/main/README.md#deleting-a-document)? I would've expected some changes to the wikihistory file too.

ddbeck committed:
Remove more <mglyph> mentions

ddbeck committed:
Remove more <mlabeledtr> mentions

ddbeck committed:
Remove more <math> attribute overflow mentions

ddbeck committed:
Remove more <mo> attributes form and largeop mentions

ddbeck committed:
Remove more <mspace> attribute linebreak mentions

ddbeck committed:
Remove more <mstyle> attribute decimalpoint mentions

ddbeck committed:
Remove more mentions of <mtable> attributes:

alignmentscope
columnwidth
equalcolumns
equalrows
groupalign
minlabelspacing
side

ddbeck committed:
Remove more <mtd> attribute groupalign mentions

ddbeck committed:
Remove more <mtr> attribute groupalign mentions

chrisdavidmills commented on files/en-us/web/mathml/element/mtable/index.html:
```diff
@@ -39,36 +37,25 @@ <h2 id="Attributes">Attributes</h2>
  <dt id="attr-columnspacing">columnspacing</dt>
  <dd>Specifies the space between table columns.</dd>
- <dt id="attr-columnwidth">{{ unimplemented_inline() }} columnwidth</dt>
  <dd>Specifies the width of table columns.</dd>
```
The corresponding `<dd>` should be removed here too, right?

ddbeck replied:
Yes! Fixed with a17039154.

chrisdavidmills commented on files/en-us/web/mathml/attribute/index.html:
```diff
@@ -303,11 +283,6 @@
-   <td>{{ MathMLElement("mtable") }}</td>
-   <td>A <a href="/en-US/docs/MathML/Attributes/Values">length value</a> specifying the minimum space between a <a href="Element/mlabeledtr" rel="internal">label</a> and the adjacent cell in the row.</td>
-  </tr>
   <tr id="minsize">
```
General comment on this file. The commit says that a mention or mentions of the `side` atttibute is removed, but I don't see anything being touched that looks like that. Just a heads up in case something was missed.

ddbeck replied:
Good catch! I double-checked and this is OK. `side` was removed from BCD, but it didn't have any consequence in the content. But I copied the list of attributes from the BCD PR and forgot to omit that one.

chrisdavidmills reviewed (changes requested):
Nice work @ddbeck . This looks nearly there now; just a couple more small comments, which should be pretty quick to wrap up.

ddbeck committed:
Remove columnwidth dd left behind in 7b2b6591c6863d0cf762994a9d091fa6acfbfc12

chrisdavidmills reviewed (approved):
I think that just about does it then. Super work here @ddbeck , thanks a lot. Let's get this merged.
